app/main/index.tsx

```
@@ -284,12 +20,15 @@ import FloatingPiece from '../../components/FloatingPiece';
 import TransitionScreen from '../../components/TransitionScreen';
 import { api } from '../../lib/api';
 import { useAppDispatch } from '../../store';
-import { createVsAI, setMode } from '../../store/gameSlice';
+import { setMode, updateFromGameObject } from '../../store/gameSlice';
+import { useRouter } from 'expo-router';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useRouter'. Consider to delete it or rename it.

'useRouter' is defined here:


(lint/suspicious/noRedeclare)